tests/rustfmt/main.rs

```diff
@@ -219,3 +219,18 @@ fn rustfmt_generates_no_error_if_failed_format_code_in_doc_comments() {
     assert!(stderr.is_empty());
     assert!(stdout.is_empty());
 }
+#[test]
+fn rustfmt_error_improvement_regarding_invalid_toml() {
+    // See also https://github.com/rust-lang/rustfmt/issues/6302
+    let args = ["--config-path", "tests/config/issue-6302.toml"];
+    let (_stdout, stderr) = rustfmt(&args);
+
+    let expected_error_message = "The file `rustfmt/tests/config/issue-6302.toml` failed to parse.\n                    Error details: invalid type: integer `2019`, expected string in `edition`";
+
+    assert!(
+        stderr.contains(expected_error_message),
+        "Unexpected error message.\nExpected:\n{}\nGot:\n{}",
+        expected_error_message,
+        stderr
+    );
+}
```

Comment:
Nit: add a space between the test and the one above it.

Here's how you can write the test to pass:

```rust
#[test]
fn rustfmt_error_improvement_regarding_invalid_toml() {
    // See also https://github.com/rust-lang/rustfmt/issues/6302
    let invalid_toml_config = "tests/config/issue-6302.toml";
    let args = ["--config-path", invalid_toml_config];
    let (_stdout, stderr) = rustfmt(&args);

    let toml_path = Path::new(invalid_toml_config).canonicalize().unwrap();
    let expected_error_message = format!("The file `{}` failed to parse", toml_path.display());

    assert!(stderr.contains(&expected_error_message));
}
```

Reply:
i have updated it  , thank you